### ISSUE TITLE
Mapping two keys to the same button

### ISSUE THREAD

**Issue by RenanBan** [fqlx/XboxKeyboardMouse#137]
I don't know if you can do this, because when i try the app doesn't let me, but make a configuration that you can choose the "Y" Xbox button to be two different keyboard keys. I guess thats understandable. I mean when you open an inventory on PUBG for example you want to use the arrow keys to sweep the itens, but in the same case the left arrow you use as a grenade button outside the inventory screen, its kinda bad to use while running and aiming to press the arrow keys. Its just an idea, i know you can map this button a lot closer to WASD, but just a tought.

**Comment by diablo2002** [comment]
hey , sorry i can't really help you out but did you find a good config for pubg ? with a good mouse mouvement ?

**Comment by RenanBan** [comment]
PUBG on Xbox one isnt made to play with a mouse and keyboard... I tune everything in game to max, disable aceleration, and tune a little bit the app x abd y sensitivity. It wont get any bettter like a rainbow six or battlefield

**Comment by diablo2002** [comment]
yup i tried multiple config for rainbow six siege but still not as good as the pc or the adapter
